dra-evolution/testdata/pools-two-nodes-dgxa100.yaml

```diff
@@ -0,0 +1,14805 @@
+- apiVersion: resource.k8s.io/v1alpha3
+  kind: ResourcePool
+  metadata:
+    creationTimestamp: null
+    name: nvidia-00-dgxa100
+  spec:
+    devices:
+    - attributes:
+      - name: uuid
+        string: GPU-c2fe2ade-955a-421f-976e-3f1dd7254234
+      - name: product-name
+        string: Mock NVIDIA A100-SXM4-40GB
+      - name: brand
+        string: Nvidia
+      - name: architecture
+        string: Ampere
+      - name: cuda-compute-capability
+        version: 8.0.0
+      - name: driver-version
+        version: 550.54.15
+      - name: cuda-driver-version
+        version: 12.4.0
+      - bool: true
+        name: mig-capable
+      - bool: false
+        name: mig-enabled
```

Comment:
I imagine there to be quite a few more attributes here eventually. Specifically around NVLINKs, PCIERoot complexes, etc. These will also need to be duplicated in each MIG device (unless we build a section for common attributes, grouped by name, that can be referenced by different devices).